pymeasure/instruments/keithley/keithley2001.py

```
@@ -0,0 +1,596 @@
+        """
+        self.write(":SENS:%s:REF:STAT 0" % self._mode_command(mode))
+
+    def acquire_reference(self, mode=None):
+        """ Sets the active value as the reference for the active mode,
+        or can set another mode by its name.
+
+        :param mode: A valid :attr:`~.Keithley2001_2002_Base.mode` name, or None for the active mode
+        """
+        self.write(":SENS:%s:REF:ACQ" % self._mode_command(mode))
+
+    def enable_filter(self, mode=None, type='repeat', count=1):
```

Ruff (0.15.6)

[error] Function argument `type` is shadowing a Python builtin

(A002)